app/Http/Controllers/Api/DepartmentsController.php

```diff
@@ -9,6 +9,7 @@
 use App\Models\Department;
 use Auth;
 use Illuminate\Http\Request;
+use app\http\Requests\ImageUploadRequest;
```

Comment:
The casing here looks a bit off, should be capitalized.

Reply:
Thanks. Fixed that.